dask/array/_shuffle.py

```diff
@@ -79,7 +87,44 @@ def shuffle(x, indexer: list[list[int]], axis):
     return Array(graph, out_name, chunks, meta=x)
 
 
-def _shuffle(chunks, indexer, axis, in_name, out_name, token):
+def _resize_other_dimensions(x: Array, longest_group, axis, chunks):
```

Comment:
```suggestion
def _resize_other_dimensions(x: Array, longest_group: int, axis: int, chunks: tuple[tuple[int, ...], ...]):
```

Reply:
chunks is currently only string, but I'll add type hints.